layers/+emacs/ibuffer/packages.el

```diff
@@ -32,7 +32,11 @@
       (evil-ex-define-cmd "buffers" 'ibuffer))
     :config
     (evilified-state-evilify-map ibuffer-mode-map
-      :mode ibuffer-mode)))
+      :mode ibuffer-mode
+      :bindings
+      "r" 'ibuffer-update
```

Comment:
We usually bind `gr` for refresh in evilified buffers, so can you bind `gr` please?